### PR TITLE
Add Open Graph elements to view game page

### PR DESCRIPTION
**Pull request by lkcampbell**
Fix for https://github.com/iftechfoundation/ifdb-suggestion-tracker/issues/12

### CODE REVIEW

**Comment by dfabulich** [comment]
How did you set up that tunnel tool?

**Comment by lkcampbell** [comment]
Used a Node.js tool called localtunnel. Here's the info:

https://samvikshana.weebly.com/blog/how-to-use-facebook-sharing-debugger-and-twitter-card-validator-with-localhost-urls

File: www/viewgame [hunk 1]
```diff
@@ -710,6 +710,22 @@ $extraHead = "<script src=\"xmlreq.js\"></script>";
...
+$extraHead .= "<meta property=\"og:url\" content=\"";
+$extraHead .= get_root_url() . "viewgame?id=$id\" />";
+$extraHead .= "<meta property=\"og:title\" content=\"$title\" />";
+if(!isEmpty($desc)) {
+    $extraHead .= "<meta property=\"og:description\" content=\"$desc\" />";
+}
+if ($hasart) {
+    $extraHead .= "<meta property=\"og:image\" content=\"";
+    $extraHead .= get_root_url() . "viewgame?coverart&amp;id=$id";
+    $extraHead .= "&amp;thumbnail=175x175\"/>";
```
Comment (dfabulich) [inline www/viewgame:726]:
```suggestion
    $extraHead .= get_root_url() . "viewgame?coverart&amp;id=$id\"/>";
```

I don't think we want to use a small thumbnail for this; we should use the biggest size we have. Stuff I've read online indicates that really big images (1200x1200) are good. https://iamturns.com/open-graph-image-size/

**Review by dfabulich — CHANGES_REQUESTED** [review]

**Comment by curiousdannii** [comment]
Maybe the title could be "TITLE by AUTHOR"?

**Comment by lkcampbell** [comment]
@curiousdannii that would certainly be easy enough to add.

I will post the idea at intfiction.org and see what people think.

**Pull request merged**